src/test/kotlin/racing/domain/NameTest.kt

```diff
@@ -0,0 +1,25 @@
+package racing.domain
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+
+class NameTest : StringSpec({
+
+    "자동차의 이름은 비어있을 수 없다" {
+        listOf(
+            "",
+            " ",
+            "  ",
+        ).forEach { emptyName ->
+            shouldThrow<IllegalArgumentException> {
+                Name(emptyName)
+            }
+        }
```

Comment:
마찬가지로 별도로 람다 변수를 명시할 필요 없이 `it`으로 변경해볼 수 있을 것 같아요. 😃
```suggestion
        ).forEach {
            shouldThrow<IllegalArgumentException> {
                Name(it)
            }
        }
```